components/uid.rst

```diff
@@ -370,6 +370,121 @@ of the ULID parameters::
+
+        use Symfony\Component\Uid\Command\GenerateUlidCommand;
+        use Symfony\Component\Uid\Command\GenerateUuidCommand;
+        use Symfony\Component\Uid\Command\InspectUlidCommand;
+        use Symfony\Component\Uid\Command\InspectUuidCommand;
+
+        return static function (ContainerConfigurator $configurator): void {
+            $services = $configurator->services()
+                ->defaults()
+                ->autowire()
+                ->autoconfigure();
+
+            $services
+                ->set(GenerateUlidCommand::class)
+                ->set(GenerateUuidCommand::class)
+                ->set(InspectUlidCommand::class)
+                ->set(InspectUuidCommand::class);
+        };
+
+Now you can generate UUIDs/ULIDs as follows (add the ``--help`` option to the
+commands to learn about all their options):
+
+.. code-block:: terminal
+
+    # generate 1 ULID with the current time as the timestamp
+    $ php bin/console ulid:generate
+
+    # generate 1 ULID with a specific timestamp
+    $ php bin/console ulid:generate --time="2021-02-02 14:00:00"
+
+    # generate 2 ULIDs and ouput them in RFC4122 format
+    $ php bin/console ulid:generate --count=2 --format=rfc4122
+
+    # generate 1 random-based UUID
+    $ php bin/console uuid:generate --random-based
+
+    # generate 1 time-based UUID with a specific node
+    $ php bin/console uuid:generate --time-based=now --node=fb3502dc-137e-4849-8886-ac90d07f64a7
+
+    # generate 2 UUIDs and output them in base58 format
+    $ php bin/console uuid:generate --count=2 --format=base58
+
+In addition to generating new UIDs, you can also inspect them with the following
+commands to show all the information for a given UID:
+
+.. code-block:: terminal
+
+    $ php bin/console ulid:inspect 01F2TTCSYK1PDRH73Z41BN1C4X
+
+     --------------------- --------------------------------------
+      Label                 Value
+     --------------------- --------------------------------------
+      Canonical (Base 32)   01F2TTCSYK1PDRH73Z41BN1C4X
+      Base 58               1BYGm16jS4kX3VYCysKKq6
+      RFC 4122              0178b5a6-67d3-0d9b-889c-7f205750b09d
+     --------------------- --------------------------------------
+      Timestamp             2021-04-09 08:01:24.947
+     --------------------- --------------------------------------
+
+    $ php bin/console uuid:inspect d0a3a023-f515-4fe0-915c-575e63693998
+
+     ---------------------- --------------------------------------
+      Label                  Value
+     ---------------------- --------------------------------------
+      Version                4
+      Canonical (RFC 4122)   d0a3a023-f515-4fe0-915c-575e63693998
+      Base 58                SmHvuofV4GCF7QW543rDD9
+      Base 32                6GMEG27X8N9ZG92Q2QBSHPJECR
+     ---------------------- --------------------------------------
+
+.. versionadded:: 5.3
+
+    The commands to inspect and generate UUIDs/ULIDs were introduced in Symfony 5.3.
```

Comment:
Don't we usually put these *before* the documentation of the feature nowadays?

Reply:
Yes. Done!